.travis.yml

```diff
@@ -55,6 +55,20 @@ jobs:
       os: linux
       jdk: openjdk13
       env: CHECK_RUST=false
+    - name: "Linux JDK EA CHECK_RUST=false"
+      os: linux
+      jdk: openjdk-ea
+      env: CHECK_RUST=false
+      before_script:
+        - if [[ "$TRAVIS_EVENT_TYPE" != "cron" ]]; then travis_terminate 0; fi
```

Comment:
Interesting, I remember the warning of "don't use exit 1", is this the right thing to use? If so, we shall update our "skip if markdown changes" script.